JS/bloggeneration.js

```diff
@@ -0,0 +1,71 @@
+
+newBlog2.setAttribute('class','blog-des');
+
+//blog3
+
+const newBlog3img = document.createElement ('img');
+
+newBlog3img.src = "images/blog-3.jpg";
+
+newBlog3img.setAttribute('class','blog-img');
+
+const newBlog3 = document.createElement('div');
+
+newBlog3.innerHTML = '<h2>Teach Your Dog To Fetch!</h2><p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Suspendisse velit dolor, mollis at lorem sed, efficitur feugiat leo. Etiam blandit tristique lacus vitae ultricies. Aliquam eros ipsum, auctor a sodales vel, accumsan ac magna. Aenean commodo faucibus metus, ut semper urna auctor eu. In volutpat sapien id lacus mollis, ac pellentesque nisi egestas. Sed ac tortor sit amet nibh fringilla venenatis. Donec vitae pulvinar diam. Curabitur finibus gravida maximus. Integer gravida sodales nisi, at viverra dui vulputate a. In malesuada ex et eros vehicula commodo. Sed porttitor eget massa vel egestas.</p><p>Integer semper eleifend commodo. Nullam vitae tristique enim, in gravida velit. Morbi eros massa, mollis id odio a, eleifend aliquam arcu. Aenean euismod tortor tellus. Ut efficitur lorem ut lacus auctor elementum. Sed luctus cursus ultricies. Phasellus at magna neque. Nam nunc tellus, maximus semper hendrerit at, interdum eu ex. Praesent vitae sapien sed quam cursus aliquam. Sed elementum, ante eget viverra convallis, metus sapien luctus elit, in interdum leo purus dignissim dui. Phasellus sed lorem nec nisi cursus feugiat. Duis mauris lacus, sollicitudin suscipit laoreet in, lacinia ac ipsum. Ut condimentum neque nec est viverra blandit ut id ligula. Sed luctus id ligula sed fermentum. </p>';
+
+newBlog3.setAttribute('class','blog-des');
+
+
+
+blogArr.push(blog1);
```

Comment:
I like the way you built out the dog blogs, but consider how you could group the individual blogs together? Including the image and the post itself into one container, then pushing that container to the  blog.